staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -610,3 +612,17 @@ func MakeRelative(path, base string) (string, error) {
 	}
 	return path, nil
 }
+
+// Deduplicate removes any duplicated values and returns a new slice, keeping the order unchanged
+func Deduplicate(s []string) []string {
+	encountered := map[string]bool{}
+	ret := make([]string, 0)
+	for i := range s {
```

Comment:
have you verified the current behavior when two configs conflict? does the latter one win? if so, need to dedupe from the back (preserve the last instance, rather than the first)